.env

```
@@ -1 +1,2 @@
-VITE_API_URL=http://localhost:5001/api
\ No newline at end of file
+VITE_API_URL=http://localhost:5001/api
+VITE_KAKAO_REST_API_KEY=03c8208960b370e6c3fdfa04651eb964
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)